packages/codd/scripts/codd.py

```
@@ -496,10 +736,55 @@ def _scope_pattern_to_regex(pattern: str) -> re.Pattern[str]:
+                index += 1
+            else:
+                out.append(_char_class_to_regex(pattern[index + 1 : end]))
+                index = end + 1
         else:
             out.append(re.escape(char))
             index += 1
-    return re.compile("".join(out))
+    try:
+        return re.compile("".join(out))
+    except re.error:
+        # `[z-a]` のような不正な文字範囲は Path.glob（fnmatch）と同様、
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)